ParseLoginUI/src/main/java/com/parse/ParseQueryAdapter.java

```diff
@@ -365,22 +378,32 @@ private void loadObjects(final int page, final boolean shouldClear) {
     // In the case of CACHE_THEN_NETWORK, two callbacks will be called. Using this flag to keep track,
     final Capture<Boolean> firstCallBack = new Capture<>(true);
 
+    runningQueries.add(query);
+
     query.findInBackground(new FindCallback<T>() {
-      @SuppressLint("ShowToast")
       @Override
       public void done(List<T> foundObjects, ParseException e) {
+        if (!runningQueries.contains(query)) {
```

Comment:
No, since we don't pass in `true` for `reportCancellation`: https://github.com/ParsePlatform/Parse-SDK-Android/blob/master/Parse/src/main/java/com/parse/ParseQuery.java#L1189